be/src/storage/seek_tuple.h

```diff
@@ -80,4 +82,23 @@ inline std::string SeekTuple::short_key_encode(size_t num_short_keys, uint8_t pa
     return output;
 }
 
+inline std::string SeekTuple::short_key_encode(size_t num_short_keys, std::vector<uint32_t> short_key_idxes,
+                                               uint8_t padding) const {
+    std::string output;
+    size_t n = std::min(num_short_keys, std::min(short_key_idxes.size(), _values.size()));
```

Comment:
Under which circumstance, the `num_short_keys` is less than `short_key_idxes`?

Reply:
> Under which circumstance, the `num_short_keys` is less than `short_key_idxes`?

If some sort columns type is float or double, we will truncate the sort keys and  the `num_short_keys` may be less than the num of`short_key_idxes`?

Reply:
Do wen need to prohibit the sort key have `float` or `double`?